operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -127,9 +127,22 @@ kiali_defaults:
 
   kubernetes_config:
     burst: 200
-    qps: 175
     cache_enabled: false
-    cache_duration: 300000000
+    cache_duration: 300
+    cache_istio_types:
+      - "DestinationRule"
+      - "Gateway"
+      - "ServiceEntry"
+      - "VirtualService"
```

Comment:
trivial - but all lists in the operator yamls do not have items indented. These items should be moved over to the left - just like you have it for the `excluded_workloads` list down below a little further. So this cache_istio_types should be:
```
cache_istio_types:
- "DestinationRule"
- "Gateway"
- "ServiceEntry"
- "VirtualService"
```

Reply:
Done. 
Yeah, I missed, the IDE doesn't modify the indentation, but when you edit first time, it proposes the pattern.